PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+<!--
```

Comment:
I would move this file in a `.github/` dir because the templates may increase a lot

Reply:
This is already the `.github` repo - if it's in the `.github` folder - will that even work? But also there's nothing else that's going to be present in this repo?

Reply:
> will that even work?

Yes, it does. I tried in [fastify/.github](https://github.com/fastify/.github)

> But also there's nothing else that's going to be present in this repo?

I suppose we will add ISSUE_TEMPLATES in future, some bot config etc..
I think a code example is better than 1000 words:

```
const orgRepo = { all directorories and files }
const dotGithubOrgRepo = { all directorories and files }

const finalRepo = Object.assign(dotGithubRepo, oneRepo)
```

So in a repo, to overwrite this default PULL_REQUEST_TEMPLATE, we should just create a `.github/PULL_REQUEST_TEMPLATE.md`

Reply:
https://github.com/pkgjs/.github/pull/1/commits/93a772b20f8f06e8403c7eb5d28471fffe60b66f